src/test/java/uk/gov/pay/api/it/CreatePaymentITest.java

```diff
@@ -57,7 +83,7 @@ public void createCardPayment() {
         String responseBody = postPaymentResponse(API_KEY, SUCCESS_PAYLOAD)
                 .statusCode(201)
                 .contentType(JSON)
-                .header(HttpHeaders.LOCATION, is(paymentLocationFor(configuration.getBaseUrl(), CHARGE_ID)))
+                .header(HttpHeaders.LOCATION, is(paymentLocationFor(configuration.getBaseUrl(), CHARGE_ID))).log().body()
```

Comment:
Did you mean to keep this `log().body()`?

Reply:
It shouldn't be there. i've deleted it.